### DISCUSSION TITLE
I have error when attache csv file.How to solve?

### DISCUSSION THREAD

**Discussion by angpao** [danny-avila/LibreChat#4304] [Troubleshooting]
### What happened?

{"level":"error","message":"Error embedding file File embedding failed.","stack":"Error: File embedding failed.\n    at uploadVectors (/app/api/server/services/Files/VectorDB/crud.js:84:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async processFileUpload (/app/api/server/services/Files/process.js:348:7)\n    at async /app/api/server/routes/files/files.js:157:5"}
{"level":"error","message":"[/files] Error processing file: File embedding failed.","stack":"Error: File embedding failed.\n    at uploadVectors (/app/api/server/services/Files/VectorDB/crud.js:95:11)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async processFileUpload (/app/api/server/services/Files/process.js:348:7)\n    at async /app/api/server/routes/files/files.js:157:5"}

### Steps to Reproduce

-

### Relevant log output

```shell
{"level":"error","message":"Error embedding file File embedding failed.","stack":"Error: File embedding failed.\n    at uploadVectors (/app/api/server/services/Files/VectorDB/crud.js:84:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async processFileUpload (/app/api/server/services/Files/process.js:348:7)\n    at async /app/api/server/routes/files/files.js:157:5"}
{"level":"error","message":"[/files] Error processing file: File embedding failed.","stack":"Error: File embedding failed.\n    at uploadVectors (/app/api/server/services/Files/VectorDB/crud.js:95:11)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async processFileUpload (/app/api/server/services/Files/process.js:348:7)\n    at async /app/api/server/routes/files/files.js:157:5"}
```

### Screenshots

<img width="438" alt="Screenshot 2567-10-02 at 08 50 12" src="https://github.com/user-attachments/assets/222971f7-2226-4a47-a968-5788b5368b78">

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Do you have the RAG API configured?

https://www.librechat.ai/docs/configuration/rag_api
